targets/esp8266/i2c_master.c

```diff
@@ -45,17 +46,19 @@
 } while(0)
 
 // For ~100khz at 80Mhz or 160Mhz:
-//#define i2c_master_wait    os_delay_us
+#define i2c_master_wait(x)    os_delay_us(x*y)
```

Comment:
merging anyway because it's ESP8266 - but this is pretty nasty :)  (using a local variable y that's not passed in as an argument)

Also you might as well just delete the commented out code - it's Git so they're still in history if anyone cares